Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,60 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+## Name
+Tips and Tricks
+
+## Description
+This is a list of useful tips and tricks to help you make the most out of SerenityOS.
+
+## General
+* When on the Desktop or in File Manager, start typing and the nearest match will be selected.
+* Bold text in context menus hints at the default behavior of a double-click.
+* Hold `Ctrl` to accelerate mouse wheel interaction with sliders and spin boxes.
+* Hold `Ctrl` while activating a menu item to prevent that menu from closing.
+* Many applications will open a compatible file if you drag-and-drop it into their window.
```

Comment:
“compatible” is not the best here imho, it begs question for compatibility with what, and itʼs not really obvious that it is about the application itself; maybe this could be slightly more precise, e.g.  
```suggestion
* Many applications will open a file they can handle if you drag-and-drop it into their window.
```
i believe that the suggestion above is somewhat wrong and you could provide a better text for that hint. 🫣

Reply:
I've tried various alternatives ("will open a _relevant_ file?") but still think the original version works the best.
I think it's obvious you can drag a `.txt` file into Text Editor.